EADModule3/WebContent/SearchStudent.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
+<style>
+td {
+	padding: 15px;
+}
+</style>
+</head>
+<body>
+	<form action="SearchStudent" method="GET">
+		<h1 align="center">Search Student By Name</h1>
+		<table align="center">
+			<tr>
+				<td>First Name :</td>
+				<td><input type="text" placeholder="First name "
+					name="first_name" required></td>
+			</tr>
+
+			<tr>
+				<td>last Name :</td>
+				<td><input type="text" placeholder="last name "
+					name="last_name" required></td>
+			</tr>
+
+			<tr>
+				<td>
+					<button type="submit" colspan="2">Submit</button>
```

Comment:
Why 2 input boxes while searching student? There should be only one input box for getting search keyword.